### PR TITLE
feat(j-s): Display message if defender has no active or completed cases

### PR DESCRIPTION
**Pull request by unakb**
[Aðgengi allra lögmanna (skráðir í lögmannaskrá) ](https://app.asana.com/0/1199153462262248/1205581325882016/f)

## What

Display message on defender case list if defender has no active and/or completed cases

## Why

For clarity

## Screenshots / Gifs
 
image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/routes/Defender/Cases/Cases.strings.ts [hunk 1]
```diff
@@ -22,4 +22,14 @@ export const defenderCases = defineMessages({
     defaultMessage: 'Afgreidd mál',
     description: 'Notaður sem titill á flipa sem sýnir afgreidd mál',
   },
+  noActiveCases: {
+    id: 'judicial.system.defender_cases:no_active_cases',
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Defender/Cases/Cases.strings.ts:26]: Consider replacing ':' with '.' to be consistent with other ids.

File: apps/judicial-system/web/src/routes/Defender/Cases/Cases.strings.ts [hunk 2]
```diff
@@ -22,4 +22,14 @@ export const defenderCases = defineMessages({
...
+    defaultMessage: 'Engin mál í vinnslu',
+    description: 'Notaður sem texti þegar engin mál eru í vinnslu hjá verjanda',
+  },
+  noCompletedCases: {
+    id: 'judicial.system.defender_cases:no_completed_cases',
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Defender/Cases/Cases.strings.ts:31]: Same as above.

**Review by gudjong — APPROVED** [review]
Minor comments

File: apps/judicial-system/web/src/routes/Defender/Cases/Cases.strings.ts (hunk 1 above)
Comment (unakb) [reply apps/judicial-system/web/src/routes/Defender/Cases/Cases.strings.ts:26]: good eye! 🙏

**Pull request merged**